x/gamm/keeper/keeper.go

```diff
@@ -77,8 +77,8 @@ func (k *Keeper) SetPoolIncentivesKeeper(poolIncentivesKeeper types.PoolIncentiv
 	k.poolIncentivesKeeper = poolIncentivesKeeper
 }
 
-func (k *Keeper) SetPoolCreationManager(poolCreationManager types.PoolCreationManager) {
-	k.poolCreationManager = poolCreationManager
+func (k *Keeper) SetPoolCreationManager(poolCreationManager types.PoolManager) {
+	k.poolManager = poolCreationManager
```

Comment:
Can we either stick to the new poolManager nomenclature, or go back to the old poolCreationManager nomenclature?